exporter/collector/integrationtest/inmemoryocexporter.go

```diff
@@ -188,6 +190,10 @@ func NewLogTestExporter(
 		logger,
 	)
 	require.NoError(t, err)
+
+	if extraConfig != nil {
```

Comment:
nit: ConfigureExporter already checks for a nil config, so this check isn't strictly necessary.  Same comment in logs_integration_test.go

Reply:
good point